src/index.html

```diff
@@ -1,13 +1,317 @@
-<!DOCTYPE html>
-<html lang="en">
+<!--#region head-->
+<!doctype html>
+<html lang="en" >
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>MyBike</title>
+    <link rel="icon" type="image/x-icon"
+    href ="images/Logo-White.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.scss">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!--#endregion-->
+  <body class = "page">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <img src="./images/WhiteBike.svg"
+              alt="Main Logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <div class="phone_block">
+                <div class="phone_block-number">
+                  <a href="tel:+123455555555"
+                  class = "phone-number"
+                  target="_blank"
+                  >
```

Comment:
Need to fix code style everywhere
```suggestion
                  <a
                    href="tel:+123455555555"
                    class="phone-number"
                    target="_blank"
                  >
```